setup.py

```diff
@@ -22,8 +27,13 @@
     tests_require=[
         'requests>=2.3.0'
     ],
+	
+    classifiers=[
+        'License :: OSI Approved :: Apache Software License',
```

Comment:
I do want to confirm, is this the correct classifier for `Apache 2.0`? It seems to be the only one that fits from the list @bp1222 commented with.

Reply:
Although, per [this](https://github.com/pypa/pypi-legacy/issues/91), Apache 2.0 is the only version that is OSI Approved so this should actually be fine

Reply:
yeah this looks good to me 👍